src/Stratis.Features.FederatedPeg.Tests/IPEndPointComparerTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Net;
```

Comment:
Move test to `Stratis.Bitcoin.Tests.P2P`

Reply:
Moved to `Stratis.Bitcoin.Utilities.Tests`.